README.md

```diff
@@ -56,7 +56,7 @@ Basic principles for operation are:
   servers.
 
 JupyterHub also provides a
-[REST API][]
+[REST API]:https://jupyterhub.readthedocs.io/en/latest/reference/rest-api.html
```

Comment:
Hi, I suggest the REST API link be embedded in "REST API".

Reply:
Thanks a lot for the suggestion I would implement it immediately

Reply:
> Thanks a lot for the suggestion I would implement it immediately

You're welcome 👍🏾